src/transformers/models/auto/modeling_auto.py

```diff
@@ -143,6 +139,10 @@
     LongformerModel,
 )
 from ..lxmert.modeling_lxmert import LxmertForPreTraining, LxmertForQuestionAnswering, LxmertModel
+
+# Add modeling imports here
+# Add modeling imports here
```

Comment:
this should probably be cleaned up (also cc @LysandreJik this looks like an error in the templates to me)